openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -1115,9 +1115,9 @@
 
   #team_concurrency_notice {
     margin: ($baseline-v/2) 0;
-    border: 1px solid $color-error;
-    background-color: $color-error-back;
-    color: $color-error;
+    border: 1px solid #ffd875;
+    background-color: #fffaed;
+    color: #5e35b1;
```

Comment:
I don't think this is actually the text color we want (it's labeled as a wireframe mock color by Figma). Also I'd check our color library for variables to use rather than rolling our own.